networkit/base.pyx

```diff
@@ -62,6 +62,8 @@ cdef class Algorithm:
 		bool
 			True if algorithm can run multi-threaded.
 		"""
+		from warning import warn
+		warn("isParallel() is deprecated and will no longer be supported.")
```

Comment:
Maybe rephrase this a bit? Like functionality will be removed in the upcoming release / Info about this can be found in the documentation.